src/app/static/src/app/components/surveyAndProgram/SurveyAndProgram.vue

```diff
@@ -85,9 +53,9 @@
 
 <script lang="ts">
     import Vue from "vue";
-    import Treeselect from '@riophae/vue-treeselect';
+    import TreeSelect from '@riophae/vue-treeselect';
```

Comment:
I was quite surprised this worked - but turns out the package does both a default and named export. We seem to use both `Treeselect` and `TreeSelect` at the moment - I have no preference.

Reply:
Me neither really - `TreeSelect` is a little easier to read